ballerina/tests/jms_session_dups_ok_ack_tests.bal

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+
+final Session dupsOkAckSession = check createSession(DUPS_OK_ACKNOWLEDGE);
+
+final MessageProducer queue6Producer = check createProducer(dupsOkAckSession, {
+    'type: QUEUE,
+    name: "test-queue-6"
+});
+final MessageConsumer queue6Consumer = check createConsumer(dupsOkAckSession, destination = {
+    'type: QUEUE,
+    name: "test-queue-6"
+});
+
+@test:Config {
+    groups: ["sessionDupsOkAck"]
+}
+isolated function testDupsOkAckWithQueue() returns error? {
+    MapMessage msg1 = {
+        content: {
+            "messageId": 1,
+            "payload": "This is the first message"
+        }
+    };
+    MapMessage msg2 = {
+        content: {
+            "messageId": 2,
+            "payload": "This is the second message"
+        }
+    };
+    MapMessage msg3 = {
+        content: {
+            "messageId": 3,
+            "payload": "This is the third message"
+        }
+    };
+    MapMessage msg4 = {
+        content: {
+            "messageId": 4,
+            "payload": "This is the fourth message"
+        }
+    };
```

Comment:
Let's remove the double quotes in here as well

Reply:
Fixed with [5f114c64f2a26a5c1b9f6bca7b54cc8cb64deb24](https://github.com/ballerina-platform/module-ballerinax-java.jms/pull/82/commits/5f114c64f2a26a5c1b9f6bca7b54cc8cb64deb24)